docs/audit_report/changes/topics/ci.yml

```diff
@@ -1,79 +1,7 @@
 title: Continuous Integration
 
-description: |
-  This contains improvements and fixes to the library's Continuous Integration
-  setup. While being important for continuous functional validation, none of
-  these patches have an immediate impact on the cryptographic functionality of
-  Botan.
```

Comment:
Maybe we can leave this general description, as I think it is basically always the same in each report? At least the this paragraph, not the one below.

Similar for docs/audit_report/changes/topics/release.yml

Reply:
Fair point. Reverted.

Note that there are other topics where the description could stay the same, that currently don't have any patches. I think, with the current setup we won't get around summarizing each topic by hand. Often it will be okay to simply take the description from the previous document.